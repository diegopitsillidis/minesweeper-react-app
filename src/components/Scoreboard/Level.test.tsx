import React from 'react';
import { render } from "@testing-library/react";
import { Level, LevelProps } from './Level';

it('Level renders correctly', () => {
    const args = {
        children: ['beginner', 'intermediate', 'expert'],
    } as LevelProps;

    const { asFragment } = render(<Level {...args} />);

    expect(asFragment()).toMatchSnapshot();
});