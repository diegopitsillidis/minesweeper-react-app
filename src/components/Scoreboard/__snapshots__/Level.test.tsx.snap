// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Level renders correctly 1`] = `
<DocumentFragment>
  <select
    class="css-cvsgr"
  >
    <option
      class="css-1ibfzol"
      value="beginner"
    >
      beginner
    </option>
    <option
      class="css-1ibfzol"
      value="intermediate"
    >
      intermediate
    </option>
    <option
      class="css-1ibfzol"
      value="expert"
    >
      expert
    </option>
  </select>
</DocumentFragment>
`;
